__tests__/app.test.js

```diff
@@ -80,6 +80,15 @@ describe("GET /api/articles/:article_id", () => {
         expect(body.msg).toBe("Invalid input");
       });
   });
+  test("status:200 responds with comment count as a property of the requested article object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+          expect(body.comment_count).toBe(11);
+          expect(body.comment_count).toEqual(expect.any(Number));
```

Comment:
The second expect here is not really necessary. You're asserting on line 88 that the count is a number by virtue of matching to 11.